core/global_constants.cpp

```diff
@@ -442,6 +443,7 @@ void register_global_constants() {
 
 	BIND_GLOBAL_ENUM_CONSTANT(JOY_SELECT);
 	BIND_GLOBAL_ENUM_CONSTANT(JOY_START);
+	BIND_GLOBAL_ENUM_CONSTANT(JOY_GUIDE);
```

Comment:
This should be after `JOY_DPAD_RIGHT` to respect the order (so `16` comes after `15` and not `11` in the docs).

Reply:
Done.